docs/source/ldap_probe/reports.rst

```diff
@@ -1,12 +1,317 @@
+
+Give the periodic task a different name, choose a schedule, and replace the
+last argument in the `Positional arguments` field with the desired performance
+degradation threshold measured in seconds and using the `float` format.
+
+Review and save.
+
+For example, in the image below, replace `warning` with `0.750` if the
+desired performance degradation threshold is 750 miliseconds.
+**Don't forget the quote signs around the float value.** 
+
+.. image:: custom_perf_report.png
+ 
+
+
+`AD` Services Network Nodes Reports
+===================================
+
+These reports are not directly related to the quality of the `AD` services
+available on the `PHSA` networks but to the quality of the data sources
+that inform this application about the network nodes providing `AD` services.
+
+These reports are generated and dispatched periodically using a configurable
+schedule.
+
+`AD` controller nodes not defined in Orion
+------------------------------------------
+
+These are `AD` nodes for which network information is not available in Orion.
+Such node are defined under `Domain Controllers not present in Orion
+</../../../admin/ldap_probe/nonorionadnode/>`__ and the information on that
+page must be maintained manually.
+
+We consider it desirable that the definitions for such `AD` nodes be created
+in Orion and, in order to facilitate such an operation, we are providing a
+periodic report with said `AD` nodes.
+
+The report is being dispatched via email using the `LDAP: non Orion AD nodes
+<../../../admin/ssl_cert_tracker/subscription/?q=LDAP%3A+non+Orion+AD+nodes>`__
+email subscription.
+
+The task that generates and executed this report is scheduled by the
+`AD controller monitoring: non Orion AD nodes report` periodic task which can be
+configured from the `periodic tasks configuration page
+</../../../admin/django_celery_beat/periodictask/>`__.
+
+`AD` controller nodes incorrectly defined in Orion
+--------------------------------------------------
+
+There are several `AD` nodes that are defined on the `Orion` server but the
+information available there is not suitable or incomplete.
+
+All the `AD` nodes that are subject to these reports must be defined as
+`domain controllers` on the `Orion` server.
+
+
+Nodes with no DNS name information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These are `AD` nodes defined on the `Orion` server for which the `DNS` `Orion`
+property is not set.
+
+The report tracking such nodes is executed automatically using the 
+`AD controller monitoring: FQDN Orion AD nodes report` periodic task
+scheduled from the `periodic tasks configuration page
+</../../../admin/django_celery_beat/periodictask/>`__.
+
+The report is dispatched via email using the `LDAP: Orion FQDN AD nodes
+</../../../admin/ssl_cert_tracker/subscription/?q=LDAP%3A+Orion+FQDN+AD+nodes>`__
+email subscription.
+ 
+
+Duplicate nodes
+^^^^^^^^^^^^^^^
+
+A subset of the `AD` nodes from above can be (and at least one **is**)
```

Comment:
point.

but we are handling all sorts of special cases in this application that are only required because the orion admin team doesn't want to play nice.
this report should be something that orion generates buy since it doesn't...